app/js/controller/list_controller.js

```diff
@@ -73,6 +73,9 @@ export default class ListController extends Controller {
       for (let manifestURL in this.list) {
```

Comment:
Yeah those events are more precise. As the list get's big that would be a much better way of doing it than the wholesale update I'm doing now. For now I think this works, but I will refactor this soon.